src/_games/demo/v1/templates/Basic.tsx

```diff
@@ -16,7 +18,24 @@ export default function Basic({ data }: { data: CardData }) {
           priority
           sizes={"(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"}
         /></div>
-      <div className="text-xl font-medium text-black">{data.text}</div>
+      <div className="text-xl font-medium text-black">
+        <ReactMarkdown components={{
```

Comment:
sigh, removing this altogether stops the warnings

Reply:
interesting, if I use markdown but remove iconFor, the error also goes away.

Reply:
no, I was wrong.

Reply:
just rendering the text with nothing else still throws the warning, so it has to do with the deps I suppose.

Reply:
it looks like if I don't import iconoir-react the warning stays away